lstchain/reco/utils.py

```diff
@@ -25,39 +25,39 @@
 from . import disp
 
 __all__ = [
-    'add_delta_t_key'
-    'alt_to_theta',
-    'az_to_phi',
-    'cal_cam_source_pos',
-    'camera_to_altaz',
-    'cartesian_to_polar',
-    'clip_alt',
-    'compute_alpha',
-    'compute_theta2',
-    'expand_tel_list',
-    'extract_source_position',
-    'filter_events',
-    'get_effective_time'
-    'get_event_pos_in_camera',
-    'impute_pointing',
-    'linear_imputer',
-    'polar_to_cartesian',
-    'predict_source_position_in_camera',
-    'radec_to_camera',
-    'reco_source_position_sky',
-    'rotate',
-    'sky_to_camera',
-    'source_dx_dy',
-    'source_side',
+    "add_delta_t_key" "alt_to_theta",
```

Comment:
there is a mistake in this definition, comma and line jump missing

Reply:
Yes, this is from an older commit. It is rectified now